core/src/main/java/org/apache/accumulo/core/metadata/TableMetadataServicer.java

```diff
@@ -77,7 +77,7 @@ public void getTabletLocations(SortedMap<KeyExtent,String> tablets)
     Text colf = new Text();
     Text colq = new Text();
 
-    KeyExtent currentKeyExtent = null;
```

Comment:
Why remove this, but not the others?

Reply:
Because this was the only one flagged by my IDE. I fixed all the ones it flagged. I didn't look for others.

Reply:
I think specifically, this removed the redundant assignment to null on line 87. If you're referring to lines 81 and 82, those assignments can't be removed, because those variables need to be initialized in subsequent lines. Line 80 did not need to be initialized because it is redundantly initialized on line 87.